fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/HavingAdapter.java

```diff
@@ -28,11 +28,15 @@ public Select build() {
     }
 
     /**
-     * Order by is the very last part of a statement
-     * 
+     * Start building the ORDER BY clause
+     * @param expressions
      * @return
      */
-    public OrderByAdapter orderBy() {
-        return new OrderByAdapter(select);
+    public OrderByAdapter orderBy(String... expressions) {
+        OrderByClause ob = new OrderByClause();
+        ob.add(expressions);
+        this.select.setOrderByClause(ob);
+        return new OrderByAdapter(select, ob);
```

Comment:
small design comment.  the OrderByClause could be converted to a builder pattern instead of overloading the toString (no action necessary, just sparking commentary)

Reply:
Yes, see earlier comment on the desire to unify the Select statement with the ExpNode visitor stuff.

Reply:
let's capture it as a backlog issue

Reply:
https://github.com/IBM/FHIR/issues/2318